mobile.css

```diff
@@ -0,0 +1,52 @@
+@media (max-width: 768px) {
+  .nav-list {
+    display: none;
+  }
+
+  .nav-bar {
+    padding: 10px;
+  }
+
+  .Introduction {
+    background: url(Assets/header.png) no-repeat center, #fff;
+    background-size: contain;
+    padding: 10px;
+  }
```

Comment:
- [ ] Kindly ensure you use the correct typo `font`, `color` as the template provided in Figma.
Your template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193144975-f5e6777b-c222-4ec6-a611-a3326ea97f56.png)
The Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193145155-0cd771d1-2330-45ef-8d45-506c4e49572e.png)